pyta.py

```diff
@@ -36,7 +48,13 @@ def check(module_name):
                                 'checkers/IO_Function_checker'])
     linter.read_config_file()
     linter.load_config_file()
-    linter.check([spec.origin])
+
+    # When module is not found, raise exception.
+    try:
+        linter.check([spec.origin])
+    except AttributeError:
+        print ("The Module '{}' could not be found in the given "
```

Comment:
@nigef  Since this is the `module not found` issue, there is no line number. In this case, the file they want to check does not exist.. so it's not their code problem..